docs/sample_config.yaml

```diff
@@ -894,6 +894,25 @@ uploads_path: "DATADIR/uploads"
 #  - matrix.org
 #  - vector.im
 
+# Handle threepid (email/phone etc) registration and password resets
+# through an identity server. Only change if you know what you are
+# doing
+#
+# IMPORTANT! This will give a malicious or overtaken identity server
+# the ability to reset passwords for your users and/or learn your
+# user's email/phone numbers! Make absolutely sure that you want to do
+# this! It is strongly recommended that these messages be sent by the
+# homeserver instead
+#
+# If this option is an empty string and SMTP options have not been
+# configured, registration by email and resetting user passwords via
+# email will be disabled
```

Comment:
This feels weird? Why isn't it just an error to supply anything but a valid thing? Or is this just meant to be handle the case of the config option is missing? In which case lets use `null`

Reply:
The code actually handles `null` and an empty string, but I agree the comment needs to be changed.